oauthproxy.go

```diff
@@ -61,6 +61,9 @@ var (
 	// ErrNeedsLogin means the user should be redirected to the login page
 	ErrNeedsLogin = errors.New("redirect to login page")
 
+	// Used to parse extra Headers
+	headerSplitRegex = regexp.MustCompile(`^([^:]*):[ ]*(.*)$`)
```

Comment:
Am I right in thinking that this will remove any space characters that exists between the colon and the value? Perhaps we should remove all whitespace characters?

Reply:
I just relooked at the http spec and tabs are indeed allowed as a separator. Added.

Reply:
The config flag immediately above this in the docs `--extra-jwt-issuers` does its key/value differentiation in a string list via `=` (`issuer=audience` in that case).

Should we align on a common separator character in our flags that intend to hold key/value pairs? (in `extra-jwt-issuers` I imagine `:` wasn't an option due to `http://...` issuers)

Reply:
I was considering it just "add these extra lines to the http header section of the request". The only reason there is any form of key=value like behavior is that the handling code expects it in that format. Otherwise I wouldn't have done the splitting at all.

But, I guess there isn't a real reason it couldn't have a different separator then ':' since we are splitting.